src/ipfs/node/repo.ts

```diff
@@ -14,7 +16,11 @@ export function create(): IPFSRepo {
     codeOrName => {
       const lookup: Record<string, BlockCodec<number, unknown>> = {
         [dagPB.code]: dagPB,
-        [dagPB.name]: dagPB
+        [dagPB.name]: dagPB,
+        [dagCBOR.code]: dagCBOR,
+        [dagCBOR.name]: dagCBOR,
+        [raw.code]: raw,
+        [raw.name]: raw,
```

Comment:
You can close this. They got in here through #389 
https://github.com/fission-codes/webnative/blob/2d0a93e4f435a51a7618ce47544fb17b391352ad/tests/helpers/in-memory-ipfs.ts#L16-L23

Reply:
Hmm, but that's another file/repo/ipfs-instance?

Reply:
Nevermind me!
Yeah, this change is in the `src/ipfs/node/repo.ts` file.
My changes are from the `tests/helpers/in-memory-ipfs.ts` file.
Whoops!